tests/core/scalar/test_summary.py

```diff
@@ -18,6 +19,7 @@
 )
 
 
+@pytest.mark.skip(reason="Needs review")
```

Comment:
why not just xfail here?

Reply:
I think with xfail, the test would still be executed, no?

Reply:
you could do `xfail(run=False)`, anyway I suggest we let it run because I saw it more often run through successfully then failing

Reply:
I've removed 9424560 again. Let's see how it will do.